contrib/integration-cli-on-swarm/agent/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
```

Comment:
I'm not sure it is possible, because `Dockerfile.worker` depends on `docker-dev:<current-branch>` so as to install the worker agent onto it.